### PR TITLE
First files (exercise 1)

### PR DESCRIPTION
**Pull request by helena-rg**

### CODE REVIEW

**Comment by hpparvi** [comment]
Great work with the code! The only two issues I'd like you to fix before merging is to revert your changes to the lectures/l05/code.f90 file and remove the binary e1 executable file.

File: lectures/l05/code.f90
Comment (hpparvi) [inline lectures/l05/code.f90:1]: This file should not be changed.

**Comment by helena-rg** [comment]
I think I revert to the previous version of the file code.f90 and remove the executable e1 already. Is it correct now?

**Comment by hpparvi** [comment]
Yes, perfect :)

**Pull request merged**